webapp/src/components/ContractLink/ContractLink.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
I like this component and I also have the same idea. But If I am not wrong, this component would be useful to have it at `decentraland-dapps`.

It is not part of the PR but we can create an issue to move it there

Reply:
I like the idea, @cazala what do you think?

Reply:
👍 

There are a bunch of Etherscan related components that we have and could live as a component or family of components like BlockDate, EtherscanLink and this one

Reply:
https://github.com/decentraland/decentraland-dapps/issues/24